packages/visualizations/src/components/MapPoi/types.ts

```diff
@@ -24,6 +25,10 @@ export interface PoiMapOptions {
     aspectRatio?: number;
     // Is the map interactive for the user (zoom, move, tooltips...)
     interactive?: boolean;
+    title?: string | undefined;
+    subtitle?: string | undefined;
+    description?: string | undefined;
+    legend?: CategoryLegend | undefined;
```

Comment:
```suggestion
    title?: string;
    subtitle?: string;
    description?: string;
    legend?: CategoryLegend;
```

Already optional so no need to explicit the `undefined` type